src/NServiceBus.AcceptanceTests/Routing/NativePublishSubscribe/When_unsubscribing_from_event.cs

```diff
@@ -107,7 +107,7 @@ public Subscriber2()
                 EndpointSetup<DefaultServer>(c =>
                 {
                     c.DisableFeature<AutoSubscribe>();
-                });
+                }, metadata => metadata.RegisterPublisherFor<Event>(typeof(Publisher)));
```

Comment:
since this tests specifies .Repeat(r => r.For<AllTransportsWithCentralizedPubSubSupport>()) there should be no need to register publishers?